pkg/scheduler/core/generic_scheduler.go

```diff
@@ -166,12 +166,6 @@ func (g *genericScheduler) Schedule(ctx context.Context, prof *profile.Profile,
 	}
 
 	// Run "prefilter" plugins.
-	preFilterStatus := prof.RunPreFilterPlugins(ctx, state, pod)
-	if !preFilterStatus.IsSuccess() {
-		return result, preFilterStatus.AsError()
-	}
-	trace.Step("Running prefilter plugins done")
```

Comment:
uhm... where are these traces exposed again?

Reply:
a log message printed at the end if the latency is higher than a specific threshold.

Reply:
https://github.com/kubernetes/kubernetes/pull/89370/files#diff-c237cdd9e4cb201118ca380732d7f361R152

Reply:
I think there are arguments for merging (pre-filter with filter and pre-score with score) and for separating them.

- Merging makes it more likely to produce a log line.
- It gives less granularity

But, I don't think a user of the log lines would care much about this granularity. It is important for us developers, but we have other tools such as profilers.

Reply:
yeah, those logs are not really meant for developers, rather for admins managing running clusters. We now have latency metrics per extension point, which offer similar fine-grained info this refactoring hides in the trace log.